src/app/contexts/user.context.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { createContext, useState } from 'react';
+
+export const userContext = createContext({} as any);
```

Comment:
how about capitalizing this to make sure the <userContext.Provider> down below looks more React? 
Or before return use const { Provider } = userContext;

Reply:
Sure